Lombiq.HelpfulExtensions/Assets/Scripts/target-blank.js

```diff
@@ -0,0 +1,20 @@
+(function initTargetBlank() {
+    function targetBlank() {
+        const links = document.querySelectorAll('a');
+        const currentHostname = window.location.hostname;
+
+        for (let i = 0; i < links.length; i++) {
+            if (!links[i].href.match(/^mailto:/) &&
+                (links[i].hostname !== currentHostname &&
+                (!links[i].href.match(/^javascript:/i)))) {
+                links[i].setAttribute('target', '_blank');
+            }
+        }
+    }
+    window.addEventListener(
+        'load',
+        () => {
+            window.setTimeout(targetBlank, 100);
```

Comment:
Why is the timeout necessary?

Reply:
Oh, I see it was in the original source.

The whole `setTimeout` call shouldn't be necessary, though.